src/payment/payment.js

```diff
@@ -9,15 +9,46 @@ import {
   cloneDeep,
 } from '../utils';
 import loadScripts from '../utils/script-loader';
+import {
+  DomElementNotFoundError,
+  PaymentElementNotCreatedError,
+} from '../utils/errors';
 
 export default class Payment {
+  _element = null;
+  _elementContainer = null;
+
   constructor(request, options, params, method) {
     this.request = request;
     this.options = options;
     this.params = params;
     this.method = method;
   }
 
+  get element() {
```

Comment:
Since this is the base class / interface, please add descriptive comments explaining the purpose of each of these methods.

Reply:
Done